spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationConfig.scala

```diff
@@ -0,0 +1,169 @@
+
+      opt[String]('f', "raw-format").required().action((value, config) => {
+        config.withRawFormat(value)
+      }).text("format of the raw data (csv, xml, parquet, fixed-width, etc.)"),
+
+      opt[String]("charset").optional().action((value, config) =>
+        config.withCharset(Some(value))).text("use the specific charset (default is UTF-8)"),
+
+      opt[String]("row-tag").optional().action((value, config) =>
+        config.withRowTag(Some(value))).text("use the specific row tag instead of 'ROW' for XML format"),
+
+      opt[String]("delimiter").optional().action((value, config) =>
+        config.withCsvDelimiter(Some(value))).text("use the specific delimiter instead of ',' for CSV format"),
+
+      opt[String]("csv-quote").optional().action((value, config) =>
+        config.withCsvQuote(Some(value)))
+        .text("use the specific quote character for creating CSV fields that may contain delimiter character(s) (default is '\"')"),
+
+      opt[String]("csv-escape").optional().action((value, config) =>
+        config.withCsvEscape(Some(value)))
+        .text("use the specific escape character for CSV fields (default is '\\')"),
+
+      // no need for validation for boolean since scopt itself will do
+      opt[Boolean]("header").optional().action((value, config) =>
+        config.withCsvHeader(Some(value))).text("use the header option to consider CSV header"),
+
+      opt[Boolean]("trimValues").optional().action((value, config) =>
+        config.withFixedWidthTrimValues(Some(value))).text("use --trimValues option to trim values in  fixed width file"),
+
+      opt[Boolean]("strict-schema-check").optional().action((value, config) =>
+        config.withFailOnInputNotPerSchema(value))
+        .text("use --strict-schema-check option to fail or proceed over rows not adhering to the schema (with error in errCol)"),
+
+      opt[String]("copybook").optional().action((value, config) => {
+        val newOptions = config.cobolOptions match {
+          case Some(a) => Some(a.copy(copybook = value))
+          case None => Some(CobolOptions(value))
+        }
+        config.withCobolOptions(newOptions)
+
+      }).text("Path to a copybook for COBOL data format"),
+
+      opt[Boolean]("is-xcom").optional().action((value, config) => {
+        val newOptions = config.cobolOptions match {
+          case Some(a) => Some(a.copy(isXcom = value))
+          case None => Some(CobolOptions(isXcom = value))
+        }
+        config.withCobolOptions(newOptions)
+      }).text("Does a mainframe file in COBOL format contain XCOM record headers"),
+
+      opt[Boolean]("cobol-is-text").optional().action((value, config) => {
+        val newOptions = config.cobolOptions match {
+          case Some(a) => Some(a.copy(isText = value))
+          case None => Some(CobolOptions(isText = value))
+        }
+        config.withCobolOptions(newOptions)
+      }).text("Specifies if the mainframe file is ASCII text file"),
```

Comment:
Validations were attached to each command-line argument in the original code. Why these validations are moved to the bottom? It made perfect sense here.